instrumentation/net/http/httptrace/otelhttptrace/constants_test.go

```diff
@@ -0,0 +1,9 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+// +build !race
```

Comment:
We will never run the tests with this tag. We should remove it.

Reply:
Done.